plugins/source/cloudflare/client/spec.go

```diff
@@ -21,5 +24,28 @@ func (s *Spec) SetDefaults() {
 	}
 }
 
+func (s *Spec) Validate() error {
+	if s.Token == "" && s.ApiKey == "" && s.ApiEmail == "" {
+		return errors.New("either api_token or api_key/api_email are required")
+	}
```

Comment:
Now that Token/Apikey/Email are required shouldn't we update the JSON schema to reflect that?

Reply:
This was done in https://github.com/cloudquery/cloudquery/pull/17605/files#diff-0dc42513fedfb6b0659c0908af5df4899e2506feab774a102f339d12c26792b4R28 see schema https://github.com/cloudquery/cloudquery/pull/17605/files#diff-ebee74c2fb7e7ba58ec5d4c118e241f72128b421c9f36c78333e21d49a118871R7